### DISCUSSION TITLE
Deployment on Netlify

### DISCUSSION THREAD

**Discussion by danwritecode** [solidjs/solid-start#257] [General]
Hey all, I tried deploying on Netlify with just npm run build with the publish directory set to dist. No luck in getting that working. Am I just too early here or am I missing something?

**Comment by ryansolid** [comment]
Yeah.. there is a netlify adapter. For CSR we should be able to make the work but if you want SSR we need to output it in a specific way and that is what the adapter does. The docs on how to use the adapters are pretty flimsy(non-existent) right now though.

 You can look at my hackernews branch for an example: https://github.com/solidjs/solid-hackernews/blob/netlify/vite.config.js

**Reply by danwritecode** [reply]
Perfect, thanks Ryan!
